nerdlandbot/translations/Translations.csv

```diff
@@ -156,4 +149,6 @@
 "empty_poncho_list","No Poncho pictures were found...","Er werden geen foto's van Poncho gevonden..."
 "poncho_pic_help","Post a random Poncho picture.","Post een willekeurige foto van Poncho."
 "poncho_hond","DOG DOG DOG","HOND HOND HOND"
+"darling","darling","schatje"
```

Comment:
'darling' and 'schatje' sound kind of creepy to me? Suggestion to change these into 'buddy' and 'vriend'

Reply:
you could also use 'mate' and 'maat' but I agree 'darling' and 'schatje' are too close

Reply:
I'll tag @bobvdvleuten in on this, as it was his idea/issue. "Schatje" is a reference to Robbe De Hert iirc.

Reply:
I agree, 'darling' and 'schatje' are a bit weird, 'mate' and 'maat' are okay for me.

Reply:
Also change 'darling' keyword to key to match the new word.